### ISSUE TITLE
Important part cannot be set within picker overlay

### ISSUE THREAD

**Issue by fritzmg** [contao/core-bundle#1093]
If you go to the file manager within the picker overlay and edit an image, you are not able to set the important part with your cursor anymore. Instead, only the input fields to manually set the important part appear.

Possibly related to #1092 ?

**Comment by ausi** [comment]
I think this can be fixed with the following changes:

```diff
--- a/src/Resources/contao/themes/flexible/src/hover.js
+++ b/src/Resources/contao/themes/flexible/src/hover.js
@@ -192,8 +192,9 @@ var Theme = {
     * Set up the profile toggle
     */
    setupProfileToggle: function() {
-       var tmenu = $('tmenu'),
-             ul = tmenu.getElement('.level_2'),
+       var tmenu = $('tmenu');
+       if (!tmenu) return;
+       var ul = tmenu.getElement('.level_2'),
            h2 = tmenu.getElement('h2');
        if (!ul || !h2) return;

```

**Comment by dmolineus** [comment]
This issue also occurs then editing content in the modal window (e.g. edit a form by clicking the edit wizard in the content element).

**Comment by ausi** [comment]
@dmolineus can you confirm that my changes also fixes your issue?

**Comment by dmolineus** [comment]
@ausi Yes, it fixes the issue.

**Comment by leofeyer** [comment]
Fixed in 21ef2c1df4c1ba9f1496f7b6fa7ed12c078ffca8. Thank you @ausi.
